src/studio/screens/console.c

```diff
@@ -3548,13 +3548,12 @@ static void onHelpCommand(Console* console)
         FOR(const struct Handler*, ptr, Handlers)
             if(strcmp(ptr->cmd, param) == 0)
             {
+                foundTopic = true;
                 ptr->handler(console);
                 break;
             }
 
-        if (foundTopic) {
-            commandDone(console);
```

Comment:
The fix on 3651 looks good (setting the flag in a situation I missed in my patch), but what makes you think this needs to be removed?  I'm not sure this change is necessary.

Reply:
`commandDone` is called twice (one more time below).

Reply:
Ah yes, I see!